azure-pipelines/start-emulator.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# This script was copy-pasted from https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/android?view=azure-devops#test-on-the-android-emulator
+# with some changes
+
+# Install AVD files
+declare -r emulator="system-images;android-$ANDROID_SDK_VERSION;google_apis;x86"
+echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install $emulator
+
+# Create emulator
+echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n testemulator -k $emulator --force
```

Comment:
it is a good practice to always enclose variable arguments into quotes `$emulator -> "$emulator"`